lambdas/build-user-identity/src/main/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandler.java

```diff
@@ -65,22 +65,26 @@ public class BuildUserIdentityHandler
     private final ClientOAuthSessionDetailsService clientOAuthSessionDetailsService;
     private final CiMitService ciMitService;
     private final CiMitUtilityService ciMitUtilityService;
+    private final VerifiableCredentialService verifiableCredentialService;
 
+    @ExcludeFromGeneratedCoverageReport // This constructor is only used in tests
```

Comment:
Required because sonarcloud takes issue with there being 8 parameters to this constructor. At the moment, 
- there is no method of reducing the number of parameters with significant rework
- this constructor is only used in tests

So I have excluded it from the coverage report

Reply:
Those are two very different reasons :)

If it's only used in tests, then it should be covered by tests - although sometimes it won't be (if it's used implicitly by the mock injection framework).

If Sonar is complaining about the code style and we want to override it, then we can add an annotation to resolve that specific issue with `@SuppressWarnings` and a reference to the issue. (Search for examples) - that way we'll still pick up other issues, rather than just blanket excluding it.

Reply:
> If it's only used in tests, then it should be covered by tests
Wouldn't the priority be production code. Or do you mean the functionality should already be covered in tests?

The `@SuppressWarnings` is a better replacement, will do that

Reply:
Ah I understand - the sonarcloud complaint was with the excessive number of parameters in the constructor (8 > 7, the threshold), as opposed to test coverage